pysingfel/detector/tests/test_autoranging_detector.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+import os
+import pytest
+
+import pysingfel as ps
+import pysingfel.gpu as pg
+import pysingfel.constants as cst
+from pysingfel.util import xp
+
+import six
+if six.PY2:
+    PSCalib = pytest.importorskip("PSCalib")
+if six.PY3:
+    psana = pytest.importorskip("psana")
+
+
+class TestAutorangingDetector(object):
+    """Test base detector functions."""
+    @classmethod
+    def setup_class(cls):
+        ex_dir_ = os.path.dirname(__file__) + '/../../../examples'
+
+        # Load beam
+        beam = ps.Beam(ex_dir_+'/input/beam/amo86615.beam')
+
+        # Load and initialize the detector
+        det = ps.PnccdDetector(
```

Comment:
why PnccdDetector? it's not an autoranging detector

Reply:
?

Reply:
What's unclear? It's an epix10ka2M detector. Why are we initializing with a PnccdDetector?

Reply:
https://github.com/chuckie82/pysingfel/pull/71/commits/20bd20bd65f00e4430e90fa564c598cdc4b2e0e2

Reply:
I don't understand what the link is for.

Reply:
20bd20b is the most recent commit, where det is initialized to Epix10kDetector.

Reply:
I see. It doesn't show here because this is now outdated code. The change is visible in the link. It only highlights the change to the import statement, so I didn't spot the change.